pipeline/splicing/Makefile

```diff
@@ -0,0 +1,68 @@
+# Host path where references should be stored - symlink tolerant
+REFERENCES = $(shell readlink -f ~/data/splicing)
+IMAGE_NAME = $(USER)-splicing-pipeline
+
+default: build references short
+
+build:
+	# Build and tag the image prefixed by user to avoid conflicts on shared machines
+	docker build -t $(IMAGE_NAME) .
+
+push:
+	# Tag and push our current docker build to dockerhub
+	docker tag $(USER)-splicing-pipeline:latest brcachallenge/splicing-pipeline:latest
+	docker push brcachallenge/splicing-pipeline:latest
+
+debug:
+	# Run the docker mapping out of local directory for development
+	docker run --rm -it \
+		--entrypoint /bin/bash \
+		--user=`id -u`:`id -g` \
+		-v `readlink -f $(REFERENCES)`:/references \
+		-v `pwd`:/app \
+	  $(IMAGE_NAME)
+
```

Comment:
seems the `unit_tests_docker` target was not so dead after all ;) If you add it back, circleci should run through. Could you also add a comment, that it is used for circleci, which I should have done anyways?

Reply:
Doh! I've put it back and tested plus comment